spring-pulsar/src/main/java/org/springframework/pulsar/annotation/PulsarReaderAnnotationBeanPostProcessor.java

```diff
@@ -233,6 +233,11 @@ else if (startMessageIdString.equalsIgnoreCase("latest")) {
 		resolveReaderCustomizer(endpoint, pulsarReader);
 	}
 
+	private void resolveSubscriptionName(MethodPulsarReaderEndpoint<?> endpoint, PulsarReader pulsarReader) {
```

Comment:
In going through this exercise I realized that we have `subscriptionName` on `@PulsarReader` which does not make sense for a `Reader`.  I am going to remove this from the Reader entirely in a subsequent commit. As such you will notice their is no new test coverage for the `@PulsarReader` since it will be removed soon.